core/src/main/resources/windows-service/jenkins-slave.xml

```diff
@@ -23,15 +23,22 @@ THE SOFTWARE.
 -->
 
 <!--
-  Windows service definition for Jenkins slave
+  Windows service definition for Jenkins agent.
+  The actual version of this definition has been moved to https://github.com/jenkinsci/windows-slave-installer-module/blob/master/src/main/resources/org/jenkinsci/modules/windows_slave_installer/jenkins-slave.xml.
```

Comment:
OK, I have found no legitimate way to use this file excepting the access to the master's classloader from plugins (why would they do it). No occurrence in OSS repos as well.

Will remove the file